game.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'item'
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(publish_date, multiplayer, last_played_at)
+    super(publish_date)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+
+  def can_be_archived?
+    super && archived
+  end
```

Comment:
- [ ] Kindly be noticed that the `can_be_archived?()` in the Game class should return true if the parent's method returns true AND if `last_played_at` is older than 2 years otherwise, it should return false.
```ruby
def can_be_archived?
      today = Date.today
      2_years_ago = Date.new(today.year - 2, today.month, today.day)

      super && @last_played < 2_years_ago
end
```

Reply:
@ShahierNashaat , thank you for noticing us. I added the function now.